docs/modules/administration/pages/installation.adoc

```diff
@@ -159,8 +162,10 @@ make install-sonarqube
 ----
 
 This will launch an instance of SonarQube.
-The script will prompt for a new admin password, set it and create an auth token to be used by the Jenkins pipelines.
-Both values can be written into `ods-configuration/ods-core.env`, which you then need to commit and push before continuing.
+The script will prompt for a new admin password, once set, an auth token that is used by the Jenkins pipelines will be generated.
```

Comment:
I think there is a typo in this sentence? Not sure how to read/understand it ...

Reply:
altered it!